services/catarse/app/views/catarse_bootstrap/user_notifier/mailer/new_friends_contributions.html.slim

```diff
@@ -15,17 +16,17 @@ h1 style="text-align:center;display:block;margin:0;padding:0;color:#3f4752;font-
                   tr
                     td style="padding-top:0;padding-right:18px;padding-bottom:9px;padding-left:18px;word-break:break-word;color:#3f4752;font-family:Helvetica;font-size:16px;line-height:150%;text-align:center" valign="top"
                       div style="text-align:center"
-                        - friends = user.contributing_friends_since_last_day(project)
+                        - friends = User.find(info[:friends_ids])
```

Comment:
**Sugestão**
Aqui poderia ser usado um `User.where(id: info[:friends_ids])`, isso faria com que essa execução não quebrasse caso o usuário não existisse mais. Exemplo: Se tu fizer: `User.find([1, 2])` e o user de ID 2 não existir, isso vai lançar um erro, mas se tu tifzer: `User.where(id: [1, 2])`, será retornado uma coleção somente com o usuário de ID 1. Mas se o comportamento do `find` for o desejado, segue o jogo.